### DISCUSSION TITLE
Could u show me an example of "Controlled Inputs" in docusaurus

### DISCUSSION THREAD

**Discussion by ghost** [facebook/docusaurus#3684] [Q&A] [answered]
output what i input as plain text

![Untitled Project](https://user-images.githubusercontent.com/30213081/98119872-100f5280-1e62-11eb-88e1-06762ce9119a.gif)

**Comment by slorber** [comment, marked as answer]
Hi, such example is not different in Docusaurus than it is in Gatsby, Next, CRA and any React app. What are you expecting exactly?

**Reply by ghost** [reply]
okay，thank u ，i found example code snippet on Gatsby website
